tests/testapp/views.py

```diff
@@ -52,6 +52,19 @@ class pagination_class(pagination.PageNumberPagination):
         page_size_query_param = 'page_size'
 
 
+class ComplexJsonFilterFieldsUserViewSet(FilterFieldsUserViewSet):
```

Comment:
PEP-8 nitpick: `ComplexJSONFilterFieldsUserViewSet` JSON (and all acronyms) should be capitalized: `Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.`